src/components/editions/index.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* eslint-disable global-require */
+
+import React from 'react';
+import clsx from 'clsx';
+import Translate from '@docusaurus/Translate';
+import Link from '@docusaurus/Link';
+import Image from '@theme/IdealImage';
+import Heading from '@theme/Heading';
+
+const Editions = [
+	{
+		name: 'Budgie',
+		image: require('@site/static/img/Budgie.jpg'),
+		url: 'budgie',
+		urlConfig: 'budgie/configuration',
+		urlTips: 'budgie/tips-and-tricks',
+		description: (
+			<Translate id="edition.budgie.description">
+			A feature-rich, luxurious desktop using the most modern technologies.
+			</Translate>
+		),
+	},
+	{
+		name: 'Plasma',
+		image: require('@site/static/img/Plasma.jpg'),
+		url: 'plasma',
+		urlConfig: 'plasma/configuration',
+		urlTips: 'plasma/tips-and-tricks',
+		description: (
+			<Translate id="edition.plasma.description">
+			A sophisticated desktop experience for the tinkerers. Simple by default, powerful when needed.
+			</Translate>
+		),
+	},
+	{
+		name: 'GNOME',
+		image: require('@site/static/img/GNOME.jpg'),
+		url: 'gnome',
+		urlConfig: 'gnome/configuration',
+		urlTips: 'gnome/tips-and-tricks',
+		description: (
+			<Translate id="edition.gnome.description">
+			A simple, streamlined desktop for more modern hardware.
+			</Translate>
+		),
+	},
+	{
+		name: 'MATE',
+		image: require('@site/static/img/MATE.jpg'),
+		url: 'mate',
+		urlConfig: 'mate/configuration',
+		urlTips: 'tips-and-tricks',
+		description: (
+			<Translate id="edition.mate.description">
+			A traditional desktop for advanced users and older hardware.
+			</Translate>
+		),
+	},
+];
+
+interface Props {
+	name: string;
+	image: string;
+	url: string;
+	urlConfig: string;
+	urlTips: string;
+	description: JSX.Element;
+}
+
+function EditionCard({name, image, url, urlConfig, urlTips, description}: Props) {
+	return (
+		<div className="col col--6 margin-bottom--lg">
+		<div className={clsx('card')}>
+		<div className={clsx('card__image')}>
+		<Link to={url}>
+		<Image img={image} alt={`${name}'s image`} />
+		</Link>
+		</div>
+		<div className="card__body">
+		<Heading as="h3"><a href={name}>{name}</a></Heading>
+		<p>{description}</p>
+		</div>
+		<div className="card__footer">
+		<div style={{textAlign: 'center'}}>
+		</div>
+		<div className="button-group button-group--block">
+		<Link className="button button--secondary" to={urlConfig}>
+		Configuration
+		</Link>
+		<Link className="button button--secondary" to={urlTips}>
+		Tips & Tricks
+		</Link>
+		</div>
+		</div>
+		</div>
+		</div>
+	);
+}
+
+export function EditionCardsRow(): JSX.Element {
+	return (
+		<div className="row">
```

Comment:
If we're doing a grid, this should be a GridItem and use Unstable_Grid from MUI. If we're doing a vertical list, use a Stack.